_posts/2023-10-23-kubeflow-1.8-release.md

```diff
@@ -0,0 +1,374 @@
+
+
+### Join the Community
+
+We would like to thank everyone for their contribution to Kubeflow 1.8, especially Daniela Plasencia for his work as the v1.8 Release Manager. As you can see, the Kubeflow community is vibrant and diverse, solving real-world problems for organizations worldwide.
+
+
+### Want to help? 
+
+Want to help? The Kubeflow community [Working Groups](https://github.com/kubeflow/community/blob/master/wg-list.md) hold open meetings and are always looking for more volunteers and users to unlock the potential of machine learning. If you’re interested in becoming a Kubeflow contributor, please feel free to check out the resources below. We look forward to working with you!
+
+
+
+* Visit our [Kubeflow website](https://www.kubeflow.org/) or [Kubeflow GitHub Page](https://github.com/kubeflow)
+* Join the [Kubeflow Slack channel](https://join.slack.com/t/kubeflow/shared_invite/enQtMjgyMzMxNDgyMTQ5LWUwMTIxNmZlZTk2NGU0MmFiNDE4YWJiMzFiOGNkZGZjZmRlNTExNmUwMmQ2NzMwYzk5YzQxOWQyODBlZGY2OTg)
+* Join the [kubeflow-discuss](https://groups.google.com/g/kubeflow-discuss) mailing list
+* Attend a [weekly community meeting](https://www.kubeflow.org/docs/about/community/)
+
+
+## Component Versions
+
+
+<table>
+  <tr>
+   <td><strong>Maintainers</strong>
+   </td>
+   <td><strong>Component Name</strong>
+   </td>
+   <td><strong>Version</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>AutoML Working Group
+   </td>
+   <td>Katib
+   </td>
+   <td><a href="https://github.com/kubeflow/katib/issues/2156">v0.16.0</a>
+   </td>
+  </tr>
+  <tr>
+   <td rowspan="9" >Notebooks Working Group
+   </td>
+   <td>Admission Webhook (PodDefaults)
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td>Central Dashboard
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td>Jupyter Web App
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td>Kubeflow Access Management API
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td>Notebook Controller
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td>Profile Controller
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td>Tensorboard Controller
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td>Tensorboard Web App
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td>Volumes Web App
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td rowspan="2" >Pipelines Working Group
+   </td>
+   <td>Kubeflow Pipelines
+   </td>
+   <td><a href="https://github.com/kubeflow/pipelines/releases/tag/2.0.2">v2.0.3</a>
+   </td>
+  </tr>
+  <tr>
+   <td>Kubeflow Pipelines Tekton
+   </td>
+   <td><a href="https://github.com/kubeflow/manifests/pull/2500">2.0.3</a>
```

Comment:
https://github.com/kubeflow/kfp-tekton/releases/tag/v2.0.3 might be a better link here